net/frr/src/opnsense/service/templates/OPNsense/Quagga/frr

```diff
@@ -16,3 +16,9 @@ start_postcmd="/usr/local/opnsense/scripts/frr/carp_event_handler"
 {% else %}
 frr_enable="NO"
 {% endif %}
+{% if helpers.exists('OPNsense.quagga.general.enablesnmp') and OPNsense.quagga.general.enablesnmp == '1' %}
+zebra_flags="-M snmp"
```

Comment:
```suggestion
zebra_flags="${zebra_flags} -M snmp"
```

if we do this here we don’t clobber unrelated flag use later, ok?

Reply:
I applied the requested change, and resynced with master.